packages/block-editor/src/components/block-list/use-block-props/use-focus-first-element.js

```diff
@@ -51,6 +51,16 @@ function useInitialPosition( clientId ) {
 	);
 }
 
+function useIsMounting() {
+	const isMounting = useRef( true );
+
+	useEffect( () => {
+		isMounting.current = false;
+	}, [] );
+
+	return isMounting.current;
```

Comment:
I think you should return the ref and not the value in case this is called in an effect in which case you'll get the refreshed value and not the one that was returned when rendered.

Reply:
Honestly, this code was written by Copilot :D I typed `useIsMounting` and it generated this for me!